### PR TITLE
add label transformer

### PR DESCRIPTION
**Pull request by 2yz**
## Changes
   
  - [x] add label transformer,  fixes https://github.com/qiniu/logkit/issues/265
 
## Reviewers

  - [ ] @wonderflow  please review

## Checklist
   
   - [ ] Rebased/mergable
   - [ ] Tests pass
   - [ ] Wiki updated

### CODE REVIEW

File: transforms/mutate/label.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		}
+		err = utils.SetMapValue(datas[i], g.Value, false, keySlice...)
+		if err != nil {
+			errnums++
+			continue
+		}
+	}
+	if err != nil {
+		g.stats.LastError = err.Error()
+		ferr = fmt.Errorf("find total %v erorrs in transform replace, last error info is %v", errnums, err)
```
Comment (wonderflow) [inline transforms/mutate/label.go]: find total %v erorrs in transform label

File: transforms/mutate/label.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		transforms.KeyFieldName,
+		transforms.KeyStageAfterOnly,
+		{
+			KeyName:      "value",
+			ChooseOnly:   false,
+			Default:      "",
+			DefaultNoUse: false,
+			Description:  "值",
+			Type:         transforms.TransformTypeString,
+		},
```
Comment (wonderflow) [inline transforms/mutate/label.go]: override说明那个也加下

Comment (2yz) [reply transforms/mutate/label.go]: override为bool类型，需要新增一个Type `transforms.TransformTypeBoolean`，并在logkit前端界面对这个类型增加处理逻辑

Comment (wonderflow) [reply transforms/mutate/label.go]: 好，我们安排下

**Comment by wonderflow** [comment]
LGTM

**Pull request merged**